databricks_blueprints/helpers.py

```diff
@@ -0,0 +1,90 @@
+import sys
+import requests
+import shipyard_utils as shipyard
+try:
+    import errors
+except BaseException:
+    from . import errors
+
+# Create Artifacts folders
+base_folder_name = shipyard.logs.determine_base_artifact_folder(
+    'databricks')
+artifact_subfolder_paths = shipyard.logs.determine_artifact_subfolders(
+    base_folder_name)
+shipyard.logs.create_artifacts_folders(artifact_subfolder_paths)
+
+
+class DatabricksClient(object):
+    "Databricks connection object for use in running queries"
+    def __init__(self, token, instance_id):
```

Comment:
I would prefer to name all `instance_id` -> `instance_url` for clarity.  With the current name, my inclination was to extract the ID from the URL only which resulted in errors.